data/thebarshablog.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Barsha Das",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Founder of **The CookieButton Project** | Co-organizer of **KubeHuddle Toronto 2023** | **Women In Web3 2022** by CoinDCX",
+  "tags": ["Open Source", "DevOps", "Web Development","Cloud Computing", "Platform Engineering", "Network Engineering"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/thebarshablog" },
+    { "icon": "FaGithub", "url": "https://github.com/thebarshablog" },
+    { "icon": "website", "url": "https://thebarshablog.github.io" }
```

Comment:
This is not a valid icon, please select one from the `Fa` or the `Si` icon sets, you can search for icons here https://linkfree.eddiehub.io/icons

Reply:
is "icon": "website" wrong?

Reply:
`website` is not a valid icon in `Fa` or `Si` icon sets, you can search for icons here https://linkfree.eddiehub.io/icons

You probably want to use something like `FaLink`

Reply:
Fixed the issue. Instead of "website", I opted for "FaLink"

Reply:
Thank you, btw did you see this in the docs somewhere as other people are using `website` also, maybe we forgot to update the docs some where?